### ISSUE TITLE
IOS - Bad JWT

### ISSUE THREAD

**Issue by kosmokoks** [b8ne/react-native-pusher-push-notifications#120]
Hello guys,

I don't receive notifications from the pusher.

error: Unrecoverable error when registering device with Pusher Beams (Reason - Bad JWT: No APNs credentials are configured for this instance)

**Comment by mandrade2** [comment]
Seems like an error on pusher side. Did you configure the APNS credentials there? https://pusher.com/docs/beams/getting-started/ios/configure-apns/

**Comment by Sulaiman122** [comment]
Same here, after uploading APN file
Do I need to create app and connect xcode to it in "app store connect" ?

**Comment by kosmokoks** [comment]
> Seems like an error on pusher side. Did you configure the APNS credentials there? https://pusher.com/docs/beams/getting-started/ios/configure-apns/

Yes, It was, I entered the keys and now it's working! Thank you very much!
